src/app/ui/landing-page/landing-page.js

```diff
@@ -22,13 +22,9 @@ let css = csjs`
     user-select:none;
   }
   .jumbotronContainer {
```

Comment:
Could you remove the class if it's empty ?

Reply:
I don't know why are those here. @ryestew can we remove those?

Reply:
I don't know why are those here. @ryestew can we remove those?

Reply:
Yes we can remove that class definition.